hubmap/query_app/queries.py

```diff
@@ -142,7 +140,15 @@ def cell_query(self, request):
         query_params = request.data.dict()
         query_params["input_set"] = request.POST.getlist("input_set")
         validate_cell_query_params(query_params)
-        pickle_hash = get_cells_list(query_params, input_set=request.POST.getlist("input_set"))
+        if (
+            query_params["input_type"] in ["dataset", "modality"]
```

Comment:
May as well check for membership `in {"dataset", "modality"}` -- Python's peephole optimizer converts that to a `frozenset` when compiling the function/module.